### ISSUE TITLE
If I change the title from the menu, it does not change with commands

### ISSUE THREAD

**Issue by arteze** [mate-desktop/mate-terminal#463]
#### Expected behaviour
If I change the title from the menu, I can then change it with the following commands.
```sh
PS1="$PS1\[\e];Pruebass\a\]"
```

#### Actual behaviour
If I change the title from the menu, when I want to change it with a command, the title is not changed.

#### Steps to reproduce the behaviour
1. Change the title from the menu
2. Run the following command to change the title (it works if I don't change the title from the options):
```sh
PS1="$PS1\[\e];Test\a\]"
```

#### MATE general version
1.28.1

#### Package version
https://abf-downloads.openmandriva.org/cooker/repository/x86_64/unsupported/release/mate-terminal-1.28.1-1-omv2490.x86_64.rpm

#### Linux Distribution
OpenMandriva Cooker 24.90 Unsupported

#### Link to bugreport of your Distribution (requirement)
https://github.com/OpenMandrivaAssociation/distribution/issues

**Comment by arteze** [comment]
It would be nice if it was in both the window name and the tab name.

**Comment by JamesR98** [comment]
In `Edit > Profile Preferences > Title and Command` is `When terminal commands set their own titles` set to something other than `Keep initial title`?

Running `PS1="$PS1\[\e];Test\a\]"` works for me (Fedora 1.28.1) with a changed option in  `Edit > Profile Preferences > Title and Command > Initial title`

**Comment by arteze** [comment]
Mate was uninstalled automatically and I get this error:
```sh
(mate-terminal:1147671): GLib-GIO-ERROR **: 23:23:19.465: Settings schema 'org.mate.interface' is not installed
```
Which Open Mandriva package should I install?

**Comment by JamesR98** [comment]
I think mate-common, can you not just reinstall mate-terminal with your package manager?

**Comment by arteze** [comment]
I don't know how to enable Unsupported mirror in dnf. I found the package and mate-terminal works now.

 - https://abf-downloads.openmandriva.org/cooker/repository/x86_64/unsupported/release/mate-desktop-schemas-1.28.2-1-omv2490.noarch.rpm

I leave a video (it's in Spanish, I'm from Argentina)

https://github.com/user-attachments/assets/7f74be53-b007-47b6-9cb3-7b443a3582d3

**Comment by JamesR98** [comment]
Ah I understand. I can confirm if you set the tab name you can no longer update either the tab name or the window title using the PS1 command. This is also the case on mate-terminal 1.26.0.

I can see this being the intended functionality if someone manually changes the title they may not want it to change again. Or they might.

**Comment by arteze** [comment]
Exactly, I want it to be able to be changed again by commands once the title has been changed manually.
